src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -84,25 +83,29 @@
         fields)
       (clear-both)
       (when-not workspace-id
-        [:div {:style {:marginBottom "1em"}}
-         [:div {:style {:fontSize "120%" :margin "1em 0"}} "Destination Workspace:"]
-         (style/create-select
-          {:value ""
-           :ref (utils/create-element-ref-handler
-                 {:store locals
-                  :key :workspace-select
-                  :did-mount
-                  #(.on (.select2 (js/$ %)) "select2:select"
-                        (fn [event]
-                          (swap! state assoc :selected-workspace
-                                 (nth workspaces-list (js/parseInt (.-value (.-target event)))))))
-                  :will-unmount
-                  #(.off (js/$ %))})
-           :style {:width 500}}
-          (sort #(compare (lower-case %1) (lower-case %2))
-                (map
-                 (fn [ws] (str (get-in ws ["workspace" "namespace"]) "/" (get-in ws ["workspace" "name"])))
-                 workspaces-list)))])
+        (let [sorted-ws-list (sort #(compare (lower-case (str (get-in %1 ["workspace" "namespace"]) "/"
+                                                              (get-in %1 ["workspace" "name"])))
+                                             (lower-case (str (get-in %2 ["workspace" "namespace"]) "/"
+                                                              (get-in %2 ["workspace" "name"]))))
```

Comment:
Does
```clojure
(sort-by #(lower-case (str (get-in % ["workspace" "namespace"]) "/"
                           (get-in % ["workspace" "name"])))
         (:workspaces-list @state))
```
work?  If so, then you could sort on namespace then name via:
```clojure
(sort-by (juxt #(lower-case (get-in % ["workspace" "namespace"]))
               #(lower-case (get-in % ["workspace" "name"])))
         (:workspaces-list @state))
```
If you keywordize this data, it can clean up a little further.